Button/Button.js

```diff
@@ -197,10 +226,13 @@ const ButtonBase = kind({
 	},
 
 	computed: {
-		className: ({animateOnRender, backgroundOpacity, highlighted, joinedPosition, selected, type, size, styler}) => styler.append(
+		className: ({animateOnRender, backgroundOpacity, highlighted, iconOnly, iconPosition, joinedPosition, selected, type, size, styler}) => styler.append(
```

Comment:
In my opinion, `joinedPosition` and `type` (with `grid` value) props couldn't be used at the same time. We might warn it if app developers.

Here is the way how to warn
```
import warning from 'warning';

	computed: {
		bgImage: ({placeholder, src}) => {
			const imageSrc = selectSrc(src) || placeholder;
			warning(imageSrc, 'Image requires that either the "placeholder" or "src" props be specified.');
			if (!imageSrc) return null;
			return placeholder && imageSrc !== placeholder ? `url("${imageSrc}"), url("${placeholder}")` : `url("${imageSrc}")`;
		},
```
https://github.com/enactjs/enact/blob/develop/packages/ui/Image/Image.js#L156

Reply:
In addition, `iconOnly` can't be used with `joinedPosition` and `type` and `highlighted` can't be used with `selected`.